### PR TITLE
Committing additions of common properties to the Device-Commons 

### PR DESCRIPTION
**Pull request by Seshan01**
In the device-schema.json namely - controlledAsset, dateLastCalibration, deviceState

To be used for the wasteWaterTreatment incubated data model

### CODE REVIEW

**Comment by Seshan01** [comment]
Hi Alberto,

This PR was prior to our decision of directly using the Device Model for the wastewater domain. Earlier, this would have been needed if we were going ahead with the 'WasterWaterDevice' entity. I think we can close/reject this PR. 

I will start a new PR to extend the Device Model with additional properties to cater to our demo case.

Sid

File: device-schema.json [hunk 1]
```diff
@@ -9,6 +9,30 @@
       "pattern": "^([[:xdigit:]]{2}[:.-]?){5}[[:xdigit:]]{2}$",
       "description": "Property. Model:'https://schema.org/Text'. The MAC address of the device."
     },
+    "ControlledAssetType": {
```
Comment (albertoabellagarcia) [inline device-schema.json:12]: use camelCase property names

File: device-schema.json [hunk 2]
```diff
@@ -9,6 +9,30 @@
...
+      "type": "array",
+      "description": "Property. Model: 'https://schema.org/Text. List of asset(s) (building, object, etc.) controlled by the device",
+      "items": {
+        "oneOf": [
```
Comment (albertoabellagarcia) [inline device-schema.json:16]: This seems to be a relationship, this was the case then it should be coded like this:
"anyOf": [
        {
          "type": "string",
          "minLength": 1,
          "maxLength": 256,
          "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
          "description": "Property. Identifier format of any NGSI entity"
        },
        {
          "type": "string",
          "format": "uri",
          "description": "Property. Identifier format of any NGSI entity"
        }
      ],

**Comment by albertoabellagarcia** [comment]
Pending changes to be implemented

**Comment by albertoabellagarcia** [comment]
This PR is outdated and most (if not all) changes have been adopted in other ways. If anything is missing please do a new PR with just this.

**Pull request closed without merging**